src/NftDetails/NftDetails.tsx

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import {useNavigation, useRoute} from '@react-navigation/native'
+import React, {useEffect, useState} from 'react'
+import {Image, StyleSheet, TouchableOpacity, View} from 'react-native'
+import {ScrollView} from 'react-native-gesture-handler'
+
+import {CopyButton, FadeIn, OfflineBanner, StatusBar, Text} from '../components'
+import {Tab, TabPanel, TabPanels, Tabs} from '../components/Tabs'
+import {useNfts} from '../hooks'
+import {NftDetailsNavigation} from '../navigation'
+
+enum VIEW_TABS {
+  OVERVIEW = 'OVERVIEW',
+  METADATA = 'METADATA',
+}
+
+type Params = {id: string}
+
+export const NftDetails = () => {
+  const {nfts} = useNfts()
+
+  const navigation = useNavigation<NftDetailsNavigation>()
+  const [activeTab, setActiveTab] = useState<VIEW_TABS>(VIEW_TABS.OVERVIEW)
+  const {id} = useRoute().params as Params
+  const nft = nfts.find((nft) => nft.id === id)
+  const stringifiedMetadata = JSON.stringify(nft, undefined, 2)
+
+  useTitle('NFT Details')
+
+  const handleFullScreenImage = () => navigation.navigate('nft-details-image', {id})
```

Comment:
prefix event handlers with "on", e.g. `onFullscreen`

inline callback is fine too